docs/Project.toml

```diff
@@ -0,0 +1,7 @@
+[deps]
+DustExtinction = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
+DustExtinction = ">=0.3.1-DEV"
+Documenter = "~0.20"
```

Comment:
:slightly_smiling_face:

Reply:
Probably this isn't clear if you don't visit the "Files changed" tab, but a line ending is missing here